docs/workforce-management-framework/topics/pro-using-demo-app.adoc

```diff
@@ -0,0 +1,103 @@
+= Using the demo application user interface
+
+The {WFM-RC-LongName} demo application consists of three components:
+
+- A mobile application for end users
+- A portal application for administrator users on a web browser
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as Workorders. Workers can view their Workorder via a mobile applicaion and administrators
+can manage workorders via the web portal.
+
+== Portal application
+
+The Portal appliction is used by a administrator to view , edit and assign Workorder and to create new Workflows.
+The Portal application is a web application. The view of workorders is restricted to administrators. 
+Onced logged in you will be presented with the following screen for admistration of Workorders and Workflows.
```

Comment:
> Onced logged in you will be presented with the following screen for admistration of Workorders and Workflows.

That's not true as developers may change default page. 
Just mentiuon that there are workforders and workflows sections (which must be there) and you should start with workflows.